src/search/search_engines/search_common.cc

```diff
@@ -80,16 +80,28 @@ shared_ptr<OpenListFactory> create_greedy_open_list_factory(
   If w = 0, we omit the h-evaluator altogether:
   we use g instead of g + 0 * h.
 */
-static shared_ptr<Evaluator> create_wastar_eval(const shared_ptr<GEval> &g_eval, int w,
+static shared_ptr<Evaluator> create_wastar_eval(const Options &options,
+                                                const shared_ptr<GEval> &g_eval, int w,
                                                 const shared_ptr<Evaluator> &h_eval) {
     if (w == 0)
```

Comment:
After adding curly braces below for the cases `w == 1` and `else`, I personally would prefer to have curly braces here as well. (Though, I always prefer to have them, so it's fine if you decide against it.)